### PR TITLE
Add HTR 10 Griffin benchmark models

### PR DESCRIPTION
**Pull request by mtnwalker**
This is a pull request to merge the Htr-10 Griffin Model with documentation.

 Refs #107

### CODE REVIEW

File: htgr/htr-10/steady/htr-10-critical.i [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+[]
+
+# ==============================================================================
+# AUXVARIABLES AND AUXKERNELS
+# ==============================================================================
+[AuxVariables]
+  [./AbsorptionRR]
+    order = FIRST
+    family = MONOMIAL
+    block = '10 11 12 13 14 15 16 17 18 20 21 22 30 31 32 40 41 42 43 44 45 46 50 52'
```
Comment (GiudGiud) [inline htgr/htr-10/steady/htr-10-critical.i:113]: can you use variable substitutions instead? eg ${fuel_blocks}
These listings are not very user friendly

File: htgr/htr-10/steady/htr-10-critical.i [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+ print_block_volume = 1
+ show_neutronics_material_coverage = 1
+[]
+
+[Outputs]
+ file_base = out_HTR-10_critical
+ interval = 1
+ exodus = true
+ csv = true
+ perf_graph = true
```
Comment (GiudGiud) [inline htgr/htr-10/steady/htr-10-critical.i:285]:
```suggestion
```

File: htgr/htr-10/steady/htr-10-critical.i [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+  [../]
+[]
+
+[Debug]
+# show_actions = 1
+ show_material_props = 1
+ check_boundary_coverage = 1
+ print_block_volume = 1
+ show_neutronics_material_coverage = 1
+[]
```
Comment (GiudGiud) [inline htgr/htr-10/steady/htr-10-critical.i:278]:
```suggestion
```
no debug output needed, this is supposed to be working

File: htgr/htr-10/steady/htr-10-full.i [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+# Power ------------------------------------------------------------------------
+
+# ==============================================================================
+# GLOBAL PARAMETERS
+# ==============================================================================
+[GlobalParams]
+  coupled_flux_groups = 'sflux_g0 sflux_g1 sflux_g2 sflux_g3 sflux_g4 sflux_g5 sflux_g6 sflux_g7 sflux_g8 sflux_g9'
+  scalar_flux = 'sflux_g0 sflux_g1 sflux_g2 sflux_g3 sflux_g4 sflux_g5 sflux_g6 sflux_g7 sflux_g8 sflux_g9'
+  #Library options include xs/htr-10-full-1RI.xml, xs/htr-10-full-393K.xml, xs/htr-10-full-523K.xml,
+  # xs/htr-10-full-ARI.xml, and xs/htr-10-full-ARO.xml
```
Comment (GiudGiud) [inline htgr/htr-10/steady/htr-10-full.i:34]: did you check all of these in?

File: htgr/htr-10/steady/tests [hunk 5]
```diff
@@ -0,0 +1,25 @@
+[Tests]
+  # Syntax Check
+  [griffin_syntax_1]
+    type = RunApp
+    input = htr-10-critical.i
+    check_input = True
+    method = 'opt'
```
Comment (GiudGiud) [inline htgr/htr-10/steady/tests:7]: add a restriction on the executable pattern, see any other folder tests

File: doc/content/htgr/htr-10/griffin_htr10_results.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+## Running an Input File
+
+If the user has access to Griffin built upon a working Moose Framework the following
+command from the shell prompt will execute the input file.
+
+```language=bash
+
+/projects/griffin/griffin-opt -i htr-10-critical.i
+
+```
```
Comment (GiudGiud) [inline doc/content/htgr/htr-10/griffin_htr10_results.md:20]: add mpi command
and 
add HPC - binary like command

File: doc/content/htgr/htr-10/griffin_htr10_results.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+compared against MCNP and Serpent using ENDFB-VI and Serpent using ENDFB-VII.r1.
+As can be seen in the table below, Griffin matches with the accepted Serpent solution very well.
+
+!table id=critical_eigen_values caption=Eigenvalues computed with different codes for the initial critical configuration. (MCNP results obtained from [!citep](IRPhEP))
+| Code  | $k_eff$  | uncertainty rel. error (pcm)  |
+| :- | :- | :- | :- |
+| MCNP (ENDFB-VI)  | 1.01190 | +/- 21 |
+| Serpent (ENDFB-VI) | 1.01025 | +/- 5.1 |
+| Serpent (ENDFB-VII.r1) | 1.00023 | +/- 2.3 |
+| Griffin TDC-SPH-Diffusion | 1.00089 | 67.3 |
```
Comment (GiudGiud) [inline doc/content/htgr/htr-10/griffin_htr10_results.md:36]: this is odd. We should get perfect preservation with SPH

File: doc/content/htgr/htr-10/griffin_htr10_results.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+
+Additionally, the flux distribution in the critical core is plotted in [htr10_flux] below.
+The effect of the upper cavity is clearly visible as the flux is almost flat within this region.
+The thermal flux peaks in the bottom conus are also visible.
```
Comment (GiudGiud) [inline doc/content/htgr/htr-10/griffin_htr10_results.md:40]: conus?

**Review by GiudGiud — CHANGES_REQUESTED** [review]
Seems something went wrong with Git

Remove the commits that revert the repo to where it was 3 months ago. Let me know if you need to talk about this

File: htgr/htr-10/steady/htr-10-full.i (hunk 4 above)
Comment (folkthom) [reply htgr/htr-10/steady/htr-10-full.i:34]: These all got merged into one master xs file.

File: doc/content/htgr/htr-10/griffin_htr10_results.md (hunk 7 above)
Comment (folkthom) [reply doc/content/htgr/htr-10/griffin_htr10_results.md:36]: From the report 
https://www.osti.gov/biblio/1468643-benchmark-analysis-htr-mammoth-reactor-physics-application

File: htgr/htr10/steady/tests [hunk 9]
```diff
@@ -0,0 +1,26 @@
+[Tests]
+  # Syntax Check
+  [griffin_syntax_1]
+    type = RunApp
+    input = htr-10-critical.i
+    check_input = True
+    executable_pattern = 'griffin*|blue_crab*'
+    method = 'opt'
```
Comment (GiudGiud) [inline htgr/htr10/steady/tests:8]: umm this probably shouldnt be here

**Comment by GiudGiud** [comment]
Cardinal failure is unrelated. 
Merging. Thank you for the contribution!

**Pull request merged**